mobile/native/src/ln_dlc/mod.rs

```diff
@@ -100,8 +100,12 @@ pub fn get_node_key() -> SecretKey {
     NODE.get().inner.node_key()
 }
 
-pub fn get_node_info() -> NodeInfo {
-    NODE.get().inner.info
+pub fn get_node_info() -> Result<NodeInfo> {
+    Ok(NODE
+        .try_get()
```

Comment:
🤔 It's kind of arbitrary to have this one be a `try_get` and not the others. It's circumstantial because this one just happens to be called earlier than the others at the moment.

I think we should investigate why the `Node` is not initialised when we expect it to be rather than papering over it. Looking at the code it does not seem possible that this would be a timing issue, because the `NODE` should be `set` before this is called. There's something strange going on.

Reply:
I absolutely would not mind if *nothing* panics, ie. we use `try_get()` everywhere. IMHO it's unacceptable to panic in any case, especially when we can't get the logs as a result.
this is just not a good UX.